holoviews/__init__.py

```diff
@@ -25,6 +25,7 @@
 from .operation import ElementOperation, MapOperation, TreeOperation # noqa (API import)
 from .element import *                                               # noqa (API import)
 from . import util # noqa (API import)
+from .element import __all__ as elements_list
```

Comment:
I'd change line 99 of ``hv.element.__init__`` to:

``return issubclass(obj, Element)``